cryptoapi_windows.go

```diff
@@ -1,7 +1,7 @@
 package certinject
 
 import (
-	"crypto/sha1"
+	"crypto/sha1" //nolint:gosec
```

Comment:
I assume this is specifically an attempt to suppress `gosec` rule G505?  If so, please use `// #nosec G505` instead, so that we don't suppress other `gosec` warnings.